### DISCUSSION TITLE
Filling In The auth.json File

### DISCUSSION THREAD

**Discussion by mwill39014** [UltimaHoarder/UltimaScraper#1080] [Q&A]
What goes in the auth_hash & auth_uniq_ fields?

Pulling up the needed values in network debugger yields these fields:

**_FIREFOX_**
csrf=xxx;
ref_src=;
fp=xxx;
auth_id=xxx;
cookiesAccepted=true;
st=xxx;
sess=xxx

**_CHROME_**
csrf=xxx;
auth_id=xxx;
__stripe_mid=xxx;
cookiesAccepted=true;
fp=xxx;
auth_uid_xxx=xxx;
sess=xxx;
st=xxx

**Comment by bigfranko** [comment]
Use the cookie helper, it gets the required info , just copy to clip board and paste in your file.  if it doesn't get it , it doesn't need it.

**Reply by mwill39014** [reply]
What/where is the cookie helper?

**Reply by fightcancer2** [reply]
> 
> 
> What/where is the cookie helper?

"When all else fails, read the instructions."

"Read the #FAQ at the bottom of this page before submitting a issue."
https://github.com/DIGITALCRIMINAL/OnlyFans

**Reply by Gamma-Velorum** [reply]
Doesn't work yo.

**Reply by M-rcus** [reply]
> Doesn't work yo.

The cookie helper extension doesn't work? Can you tell me what's not working and what browser/OS you're using?

**Reply by 7zed** [reply]
> > Doesn't work yo.
> 
> The cookie helper extension doesn't work? Can you tell me what's not working and what browser/OS you're using?

the auth hash and auth uniq and x_bc are empty on the cookie helper. how to fix? i use mozilla

**Reply by M-rcus** [reply]
> > > Doesn't work yo.
> > 
> > 
> > The cookie helper extension doesn't work? Can you tell me what's not working and what browser/OS you're using?
> 
> the auth hash and auth uniq and x_bc are empty on the cookie helper. how to fix? i use mozilla

They are empty because they're supposed to be.  
`x_bc` will be filled by the _ripper software_ if it ever becomes necessary (at the moment, it seems not to be?)  
`auth_hash` and `auth_uniq_` don't seem to be used anymore - not by the website itself (nor by the ripping software), but they're left "as is" because I'm not sure how the ripping software would react if they weren't included.  
`auth_uniq_` was used in the past for accounts with 2 factor authentication enabled, but it seems now it's no longer being set and it's not necessary for the ripping software either.

Did you try the `auth.json` with the data from the cookie helper extension at all, or just assume it didn't work because it was missing values?

**Reply by 7zed** [reply]
> > > > Doesn't work yo.
> > > 
> > > 
> > > The cookie helper extension doesn't work? Can you tell me what's not working and what browser/OS you're using?
> > 
> > 
> > the auth hash and auth uniq and x_bc are empty on the cookie helper. how to fix? i use mozilla
> 
> They are empty because they're supposed to be.
> `x_bc` will be filled by the _ripper software_ if it ever becomes necessary (at the moment, it seems not to be?)
> `auth_hash` and `auth_uniq_` don't seem to be used anymore - not by the website itself (nor by the ripping software), but they're left "as is" because I'm not sure how the ripping software would react if they weren't included.
> `auth_uniq_` was used in the past for accounts with 2 factor authentication enabled, but it seems now it's no longer being set and it's not necessary for the ripping software either.
> 
> Did you try the `auth.json` with the data from the cookie helper extension at all, or just assume it didn't work because it was missing values?

sorry i worked that out. the script worked the first time. https://github.com/DIGITALCRIMINAL/OnlyFans/discussions/1166
second time the cmd had errors

**Comment by LunarPenguin** [comment]
not all fields show up for all users so if there is a field which you don't see in the debugger, leave it blank. The `auth_uniq_` field is called the `auth_uniq_<your_auth_id_here>` field in the debugger.
